direct-query-core/src/main/java/org/opensearch/sql/directquery/rest/model/ExecuteDirectQueryResponse.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.directquery.rest.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ExecuteDirectQueryResponse {
+  private String queryId;
+  private String result;
+  private String sessionId;
```

Comment:
Is this needed if we have the queryId? What benefit is there to add another tracker if we can just use queryId everywhere instead.